src/components/user/password_change/PasswordChange.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Form } from 'react-bootstrap';
+
+/**
+ * This component simply displays a users information.
```

Comment:
This comment should be updated

Reply:
Ahh true. Thanks for catching this. Was recycling another components structure here.